doc/rados/configuration/osd-config-ref.rst

```diff
@@ -442,7 +442,15 @@ Operations
 
 ``osd recovery op priority``
 
-:Description: The priority set for recovery operations, if not specified by the pool's ``recovery_op_priority``.
+:Description: The priority set for recovery operations, if not specified by the
+              pool's ``recovery_op_priority``.  Setting a lower value decreases
```

Comment:
Can we make this more concise? Two things to highlight here:

- these are priorities used by messages associated with recovery
- by default, client ops(which use osd client op priority = 63) are prioritized over recovery ops (which use osd recovery op priority = 3)

Reply:
Sure.  With contradictory views on the list, and especially with other documents beyond our control giving the wrong information, I was aiming to remove any ambiguity.  I've reworded this and the client op priority above, PTAL.